docs-src/guide-rpc-server-api.scrbl

```diff
@@ -0,0 +1,333 @@
+#lang scribble/manual
+@(require "lib.rkt")
+
+@title[
+  #:version reach-vers
+  #:tag     "guide-rpc-server-api"
+  ]{The Reach RPC server API and how to use it}
+
+The Reach RPC server's design is deliberately lean and simple. In fact, it
+mostly just re-exports everything offered by the standard library via HTTPS
+under a corresponding @urlin{/stdlib/<function name>} route. Arguments and
+their order remain unchanged.
+
+@;TODO better example(?)
+For example, rather than leveraging the standard library directly like so:
+@js{ const balance = stdlib.formatCurrency('100000000000000000000', 18); }
+
+...an equivalent RPC request is issued, where the payload is a JSON-encoded
+array which represents ordered arguments to the target function:
+@js{ POST /stdlib/formatCurrency ['100000000000000000000', 18] }
+
+Likewise, Reach RPC @italic{@tech{frontends} and client libraries} are
+correspondingly lean and simple. Unless you're constructing a new client
+library (@seclink["guide-rpc-connect"]{more on this later}) for a language
+which doesn't yet have one available, you won't issue these RPC requests
+directly; you'll instead send such requests via a thin wrapper which has
+already been provided for you.
+
+@py{
+  # In this example we're using the Python RPC client library
+  balance = rpc('/stdlib/formatCurrency', '100000000000000000000', 18)
+}
+
+@italic{The remainder of this section introduces concepts discretely and with
+short code snippets. If you'd prefer to see fully-realized examples of
+@tech{RPC} @tech{frontends} in their entirety, it's recommended to browse the
+@urlin{client-<language>} directories @reachexlink["tut-6-rpc" "here"].}
+
+
+@;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
```

Comment:
You have a hard job here... you are simultaneously talking about the server API /and/ the Python client implementation. It might be easier to "reference-ize" it and have a boring listing of the server routes, then a section on each language client package with their own little notes, and then finally a tutorial section that actually uses python for the tutorial